src/lib/services/openapi-service.ts

```diff
@@ -58,10 +59,36 @@ export class OpenApiService {
     useErrorHandler(app: Express): void {
         app.use((err, req, res, next) => {
             if (err && err.status && err.validationErrors) {
-                res.status(err.status).json({
-                    error: err.message,
-                    validation: err.validationErrors,
+                const errors = err.validationErrors.map((validationError) => {
```

Comment:
Lines 62 to  90 are getting quite involved. I'd extract them into a function and unit test in isolation.

Reply:
Aye, I agree with that.

Reply:
Addressed in 2d4aaf9cb22c4e4f6cc8dec958696a9998437c9f